tests/local_clean_urls.php

```diff
@@ -202,6 +212,7 @@ public function test_local_cleanurls_simple() {
         $url = 'http://www.example.com/moodle/user/view.php?id=' . $this->staff->id . '&course=' . $this->course->id;
         $murl = new moodle_url($url);
         $clean = $murl->out();
+        var_dump($clean);
         $this->assertEquals('http://www.example.com/moodle/course/'.$this->course->shortname.'/user/' . $this->staff->username,
```

Comment:
$this->course->shortname should be wrapped in urlencode($this->course->shortname)